src/main/webserver/index.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
We already have a index.js for the client-side javascript file for the home page. Therefore I think naming this index.js is a bad idea. Since it is the starting point of the webserver maybe call it server.js?